### PR TITLE
update llm latency benchmark.

### PR DESCRIPTION
**Pull request by lalalapotter**
## Description

This PR is to update llm latency benchmark.

### CODE REVIEW

**Review by hkvision — APPROVED** [review]

File: python/llm/dev/benchmark/all-in-one/run-spr.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
-
+source bigdl-nano-init -c
```
Comment (jason-dai) [inline python/llm/dev/benchmark/all-in-one/run-spr.sh:2]: Do we need `nano` besides `bigdl-nano-init`? If not, we can simple copy the script to `llm` and eliminate the dependency on `nano`

Comment (lalalapotter) [reply python/llm/dev/benchmark/all-in-one/run-spr.sh:2]: Sure, maybe we can add a script named `bigdl-llm-init` and exclude jemalloc related part.

File: python/llm/dev/benchmark/all-in-one/run-spr.sh [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+export OMP_NUM_THREADS=48
+export TRANSFORMERS_OFFLINE=1
 
 # set following parameters according to the actual specs of the test machine
 numactl -C 0-47 -m 0 python $(dirname "$0")/run.py
```
Comment (jason-dai) [inline python/llm/dev/benchmark/all-in-one/run-spr.sh:7]: maybe accept core# as a parameter to automatically set `OMP_NUM_THREADS` and `numactl` values

File: python/llm/dev/benchmark/all-in-one/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # All in One Benchmark Test
 All in one benchmark test allows users to test all the benchmarks and record them in a result CSV. Users can provide models and related information in `config.yaml`.
 
-Before running, make sure to have [bigdl-llm](../../../README.md) installed.
+Before running, make sure to have [bigdl-llm](../../../README.md) and [bigdl-nano](../../../../nano/README.md) installed.
```
Comment (jason-dai) [inline python/llm/dev/benchmark/all-in-one/README.md:4]: do we really need `nano`?

File: python/llm/dev/benchmark/all-in-one/run-spr.sh (hunk 2 above)
Comment (lalalapotter) [reply python/llm/dev/benchmark/all-in-one/run-spr.sh:7]: `numactl -C` not only stands for number of cores but also index of cores, so parameter core# can only work for `OMP_NUM_THREADS`. Will try to find a simpler way to set the variables.

File: python/llm/dev/benchmark/all-in-one/README.md (hunk 3 above)
Comment (lalalapotter) [reply python/llm/dev/benchmark/all-in-one/README.md:4]: I think if we add `bigdl-llm-init` for BigDL-LLM, there should be no need for nano.

File: python/llm/dev/benchmark/all-in-one/run-spr.sh (hunk 2 above)
Comment (jason-dai) [reply python/llm/dev/benchmark/all-in-one/run-spr.sh:7]: always use 0-`core#-1` for numactl

Comment (lalalapotter) [reply python/llm/dev/benchmark/all-in-one/run-spr.sh:7]: In this way, we could not specify the socket of CPU. For example, if we are going to use second socket, the command should be `numactl -C 48-95 ...`

Comment (jason-dai) [reply python/llm/dev/benchmark/all-in-one/run-spr.sh:7]: I don't think there is such need in bechnmarking

Comment (lalalapotter) [reply python/llm/dev/benchmark/all-in-one/run-spr.sh:7]: Sure, then we will add parameter core# and automatically set `OMP_NUM_THREADS` and `numactl` values

File: python/llm/dev/benchmark/all-in-one/README.md (hunk 3 above)
Comment (hkvision) [reply python/llm/dev/benchmark/all-in-one/README.md:4]: We can add the nano env into llm in a later PR.

**Pull request merged**